mobile/mobile-core/android-process-service.ts

```diff
@@ -271,6 +278,19 @@ export class AndroidProcessService implements Mobile.IAndroidProcessService {
 
 		return result;
 	}
+
+	private tryAttachToProcessExitSignals(): void {
+		this.$processService.attachToProcessExitSignals(this, () => {
+			return (() => {
```

Comment:
no need of future here - no asyn methods (check my comment on IListener interface)